client/extensions/hello-dolly/state/selectors.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Internal Dependencies
+ */
+import lyrics from './lyrics';
+
+export function getCurrentLyric( reduxState ) {
+	const state = reduxState.helloDolly;
+
+	return lyrics[ state.lyricIndex ];
```

Comment:
This makes sense, I've just started getting in the habit of currying bits of data along the way.